lib/push.js

```diff
@@ -218,7 +218,7 @@ module.exports = function (log, db, config) {
         }
       }))
       var options = { data: data, TTL: TTL_DEVICE_DISCONNECTED }
-      return this.pushToDevice(uid, idToDisconnect, 'deviceDisconnected', options)
```

Comment:
The `idToDisconnect` isn't actually used outside the push service in this file. Calling `pushToAllDevices` still sends the id in the payload as it does db read for the device ids and passes that to `pushToDevices`.

`pushToDevice` simply calls `pushToDevices` with an individual id wrapped in an array.